index.js

```diff
@@ -1,19 +1,26 @@
 /* Generates a sitemap */
+const { env } = require('process')
+
 const makeSitemap = require('./make_sitemap')
 
+const getBuildDir = ({ inputs, constants }) => {
```

Comment:
As a result of code complexity rule

Reply:
Yes, that's actually probably a good thing to split this function.